src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2523,3 +2535,10 @@ def create_devops_build(cmd, functionapp_name=None, organization_name=None, proj
                                                                  organization_name, project_name, repository_name,
                                                                  overwrite_yaml, allow_force_push, use_local_settings)
     return azure_devops_build_interactive.interactive_azure_devops_build()
+
+
+def _configure_default_logging(cmd, rg_name, name):
```

Comment:
Make sure do it in the follow-up PR that you should do nothing if the diagnostics are already configured

Reply:
Just want to clarify this- the API will not change anything if these specific logs are turned on. 
With regards to a scenario where a user might have Blob stoarage enabled & we are enabling fileSystem, then this will not turn off blobStorage one. On our end we allow both of these to be enabled. For logstreaming to display messages on console the FileSystem & weblogging should be enabled. If you still think this might be a problem i will file an issue to track this - especially we get some feedback around this.